pkgs/development/python-modules/pynput/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildPythonPackage, fetchPypi, sphinx, setuptools-lint, xlib }:
+
+buildPythonPackage rec {
+  pname = "pynput";
+  version = "1.6.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "16h4wn7f54rw30jrya7rmqkx3f51pxn8cplid95v880md8yqdhb8";
+  };
+
+  propagatedBuildInputs = [ sphinx setuptools-lint xlib ];
```

Comment:
sphinx is usually meant for documentation, should probably go in nativeBuildInputs

Reply:
You're right, I put it in `nativeBuildInputs` now